android-components/components/feature/push/src/main/java/mozilla/components/feature/push/AutoPushFeature.kt

```diff
@@ -431,3 +371,35 @@ data class PushConfig(
     val serviceType: ServiceType = ServiceType.FCM,
     val disableRateLimit: Boolean = false,
 )
+
+/**
+ * Helper function to get the corresponding support [BridgeType] from the support set.
+ */
+@VisibleForTesting
+internal fun ServiceType.toBridgeType() = when (this) {
+    ServiceType.FCM -> BridgeType.FCM
+}
+
+fun Protocol.toRustHttpProtocol(): PushHttpProtocol {
```

Comment:
Can this function be internal or private? I also think `detekt` might be unhappy unless a kdoc is added.